### DISCUSSION TITLE
[Request] Keep system font

### DISCUSSION THREAD

**Discussion by xerta555** [community/community#38000] [Mobile]
Hi

It could be possible to add an option to keep the system font, or let the choice to the user to use the Github specific font, or use the system font for the Github mobile app?

I have check on the last stable and on the last Beta version of the Android app, i don't see this option.

Thanks in advance.

**Comment by stevepopovich** [comment]
Hi Rom, thanks for the feedback. We made a decision to force some of the fonts in the app for layout purposes, but maybe we should revisit it. I will bring this to the team. Thanks!

**Reply by xerta555** [reply]
> Hi Rom, thanks for the feedback. We made a decision to force some of the fonts in the app for layout purposes, but maybe we should revisit it. I will bring this to the team. Thanks!

Thank you, i sincerely think it would be more prudent to give users the choice through an option in the application settings.

If the developers want to, apply Github's font by default & display a warning if they have the faith and patience to code a warning in the app to say "hey be careful with the layout it could crash everything", even though i know in advance that won't be the case.
